packages/integration-tests/test-projects/data-storage-auth-with-triggers-cjs/update-1/data/resource.ts

```diff
@@ -1,4 +1,4 @@
-import { defineData } from '@aws-amplify/backend-graphql';
+import { defineData } from '@aws-amplify/backend';
```

Comment:
it seems that we're missing assertion somewhere in that update logic
@Amplifiyer 

if we do, we should at least cut GH issue.

Reply:
done: https://github.com/aws-amplify/amplify-backend/issues/747